core/templates/pages/skill-editor-page/services/skill-editor-state.service.ts

```diff
@@ -86,7 +88,11 @@ export class SkillEditorStateService {
     this._setSkill(skill);
   };
 
-  private _updateGroupedSkillSummaries = (groupedSkillSummaries) => {
+  private _updateGroupedSkillSummaries = (
+      groupedSkillSummaries: {
+      [topicName: string]: SkillSummaryBackendDict[];
+    }
```

Comment:
```suggestion
      groupedSkillSummaries: {
        [topicName: string]: SkillSummaryBackendDict[];
      }
```

and maybe put this into a type variable too